staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -793,6 +793,20 @@ type KubeletConfiguration struct {
 	// Default: false
 	// +optional
 	KernelMemcgNotification bool `json:"kernelMemcgNotification,omitempty"`
+	// registerNode enables automatic registration with the apiserver.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact the ability of the Kubelet to interact with the apiserver
+	// Default: true
+	// +optional
+	RegisterNode *bool `json:"registerNode,omitempty"`
```

Comment:
It does not matter to have pointer on external here, since the field merges happen in the internal values.